generator/app.go

```diff
@@ -686,10 +785,22 @@ func writeYaml(data interface{}, path string) error {
 }
 
 func main() {
+
+	// Fetch KEPs and cache them in the keps variable
+	err := fetchKEPs()
+	if err != nil {
+		os.Exit(1)
```

Comment:
```suggestion
		log.Fatal(err)
```
Can we log these out as fatal for diagnosis?